src/MEDS_transforms/extract/extract_code_metadata.py

```diff
@@ -364,6 +364,10 @@ def main(cfg: DictConfig):
     OmegaConf.save(event_conversion_cfg, partial_metadata_dir / "event_conversion_config.yaml")
 
     events_and_metadata_by_metadata_fp = get_events_and_metadata_by_metadata_fp(event_conversion_cfg)
+    if not events_and_metadata_by_metadata_fp:
+        logger.info("No _metadata blocks in the event_conversion_config.yaml found. Exiting...")
+        return
```

Comment:
Given your comment on #125, I'll calrify that the symlink should symlink `cfg.stage_cfg.reducer_output_dir` to point to `cfg.stage_cfg.metadata_input_dir`

Reply:
Now that #110 is solved, I think we can skip the symlinking as there are no metadata stages before this in the extraction pipeline. It may raise issues in the future, but I think it is acceptable for now.